universum/configuration_support.py

```diff
@@ -13,6 +16,47 @@
 skip_attributes = {"children", "critical", "skip_numbering_level"}
 
 
+ProjectConfigKeyType = Literal["name", "command", "environment", "artifacts", "report_artifacts",
+                               "artifact_prebuild_clean", "directory", "critical", "background", "finish_background",
+                               "code_report", "pass_tag", "fail_tag", "if_env_set"]
+
+
+class ProjectConfig(TypedDict, total=True):
+    """
+    :param name: Specifies the name of the configuration.
+    :type name: str
+    The name is used as the title of the build log block corresponding to the build of this configuration. It is
+    also used to generate name of the log file if the option for storing to log files is selected. A project
+    configuration can have no name, but it is not recommended for aesthetic reasons. If several project
+    configurations have the same names, and logs are stored to files (see --launcher-output / -lo command-line
```

Comment:
``--launcher-output`` / ``-lo`` still looks ok in docstrings, and might look way better for generated documentation

Reply:
I don't follow this comment. Would you like me to change something?

Reply:
Yes, I meant changing '-quotes to `-quotes exactly for reasons demonstrated in comment above.

Reply:
I just fully copied descriptions from 'configuring.rst' instead